pkg/controllers/termination/eviction.go

```diff
@@ -49,12 +49,12 @@ func NewEvictionQueue(ctx context.Context, coreV1Client corev1.CoreV1Interface)
 
 		coreV1Client: coreV1Client,
 	}
-	go queue.Start(ctx)
+	go queue.Start(logging.WithLogger(ctx, logging.FromContext(ctx).Named("eviction")))
```

Comment:
Can we name this `termination.eviction` or `termination/eviction`? In log lines we see `controller.termination` for other termination related behaviors, but since this lies only touches termination behavior, I think it'd make sense to notate it as a subcontroller of termination.

Reply:
I played with this and it changes the indentation, which makes the logs harder to read. I think it's probably 6-of-1-half-a-dozen-of-the-other. We can always change this too, so keeping as is if users get confused.